src/streams/logs.ts

```diff
@@ -0,0 +1,69 @@
+import { Readable } from 'stream';
+import { listOnePageLogResources } from '../api/logs';
+import { LogApiResource, Sid, GotClient } from '../types';
+import { LogsConfig } from '../types/logs';
+
+export class LogsStream extends Readable {
+  _buffer: Array<LogApiResource>;
```

Comment:
can you prefix everything that starts with `_` with `private`? It's not gonna change the compiled output but it will mark them for the resulting typescript types as `private`. So:
```ts
private _buffer: Array<LogApiResource>
```